src/main/java/com/anthonymace/contacttasks/api/v1/resource/TaskResource.java

```diff
@@ -0,0 +1,66 @@
+package com.anthonymace.contacttasks.api.v1.resource;
+
+import com.anthonymace.contacttasks.services.TasksService;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+@RestController
+public class TaskResource {
+
+    private TasksService tasksService = new TasksService();
```

Comment:
If this is Spring-ish, can't this be passed in through a constructor/autowired vs. having to new it up?

Reply:
Probably not an issue for something this lightweight, but I'm thinking of considerations for larger scale systems where this service might be re-used and having to consider singletons, beans, etc.